Policies/github-terms-of-service.md

```diff
@@ -46,6 +46,7 @@ Effective date:
 4. “The User,” “You,” and “Your” refer to the individual person, company, or organization that has visited or is using the Website or Service; that accesses or uses any part of the account; or that directs the use of the account in the performance of its functions. A User must be at least 13 years of age. Special terms may apply for business or government accounts (See [Section B(4): Additional Terms](#4-additional-terms)).
 5. “GitHub,” “We,” and “Us” refer to GitHub, Inc., as well as our affiliates, directors, subsidiaries, contractors, licensors, officers, agents, and employees.
 6. “Content” refers to content featured or displayed through the Website, including without limitation text, data, articles, images, photographs, graphics, software, applications, designs, features, and other materials that are available on the Website or otherwise available through the Service. "Content" also includes Services. “User-Generated Content” is Content, written or otherwise, created or uploaded by our Users. "Your Content" is Content that you create or own. “Paid Content” is Content only available to Users who are participating in a payment plan, including private repositories.
+7. “Free Licence” refers to a Free / Open Source / Open Knowledge / Ⓕ Copyfree licence. For practicality reasons, the set of Free Licences is comprised of the licences listed at [ifrOSS](http://www.ifross.org/en/license-center), [Debian](https://wiki.debian.org/DFSGLicenses), [OKFN](http://opendefinition.org/licenses/), [OSI](https://opensource.org/licenses/alphabetical), [FSF](https://www.gnu.org/licenses/license-list.html), [the Copyfree Initiative](http://copyfree.org/standard/licenses), and/or the [list of licences approved for Free Cultural Works](http://freedomdefined.org/Licenses), explicitly excluding those listed on these sites as not Free (such as those under the headings “B. Other Software Licenses”, “D. Other Licenses for Fair Use of Immaterial Goods”, “DFSG-incompatible”, “Non-Conformant Licenses”, “Nonfree Software Licenses”, “Nonfree Documentation Licenses”, “Licenses for Works stating a Viewpoint”, “Invariant sections” in the linked lists).
```

Comment:
The list of lists is not perfect. However, it is a start, it doesn’t contain any false positives, it doesn’t need GitHub to finally disclose what rights their lawyers think they **precisely** need, and it fixes the problem the new ToS introduced in February for most OSS projects.

Do this now, improve upon it in the background in parallel so a future ToS update will see your improvements. (I’m all for the idea of requirement-based, but you also have to state what those licences can _not_ require from users or GitHub, *and* you *will* have to list approved licences somewhere because putting this down on every user will be ridiculously hard.) Also, note that forks do *not* get “free redistribution”, only inside GitHub, by the default licence.

For now, this change will make a couple of licences “first class citizens”, but for OSS stuff, that’s okay.

The organisations are known to either not change the criteria, or only in the same spirit. I think we can believe that anything that passes one set of those criteria and does not fail others’ sets in those list is “good enough” to be promoted to first-class citizen and thus explicitly excluded from the extra licence clause.